cli/cmd/alert_list.go

```diff
@@ -242,17 +251,29 @@ func listAlert(_ *cobra.Command, _ []string) error {
 		return errors.Wrap(err, msg)
 	}
 
+	// filter fixable
+	alerts := listResponse.Data
+	if alertCmdState.Fixable {
+		alerts, err = filterFixableAlerts(alerts, getRemediationTemplateIDs)
+		if err != nil {
+			return errors.Wrap(err, "unable to filter by alert fixability")
+		}
+	}
+
 	if cli.JSONOutput() {
-		return cli.OutputJSON(listResponse.Data)
+		return cli.OutputJSON(alerts)
 	}
 
-	if len(listResponse.Data) == 0 {
+	if len(alerts) == 0 {
 		cli.OutputHuman("There are no alerts in your account in the specified time range.\n")
 		return nil
 	}
-	renderAlertListTable(listResponse.Data)
+	renderAlertListTable(alerts)
 
 	// breadcrumb
 	cli.OutputHuman("\nUse 'lacework alert show <alert_id>' to see details for a specific alert.\n")
+	if alertCmdState.Fixable {
+		cli.OutputHuman("Use 'lacework remediate alert <alert_id>' to fix a specific alert.\n")
```

Comment:
are you purposely printing both breadcrumbs? Wouldn't you like to be more prescriptive and chose one or the other? Thinking out-loud here.

Reply:
Yes, I think both breadcrumbs is what we want if using `--fixable`.  `lacework alert show` is still valid to see details and `lacework remediate alert` if you wanna fix.

Reply:
Fixed.